ws-nextjs-app/pages/[service]/send/[id]/FormField/styles.ts

```diff
@@ -57,6 +57,7 @@ export default {
       padding: `${spacings.FULL}rem`,
       ...fontVariants.sansRegular,
       ...fontSizes.pica,
+      [mq.HIGH_CONTRAST]: { borderColor: 'canvasText', outline: '0' },
```

Comment:
Are you able to change the name of this to 'mq.FORCED_COLOURS' in this PR @HarveyPeachey?

Reply:
I'd rather have it in a seperate PR for seperation of concern due to it being changed in multiple places. I can fire up a quick PR with the change in it

Reply:
Thanks Harvey 😄